dashboard/projects/views.py

```diff
@@ -204,6 +224,18 @@ def project_basic_form_save(request, project_id):
     form.save()
     return render(
         request,
-        "projects/partial_project_basics.html",
+        "projects/partial_project_detail_basics.html",
         {"basics_form": form, "project": instance},
     )
+
+
+# admin methods
+
+def admin_recalculate_all_levels(request):
```

Comment:
```suggestion
@staff_member_required
def admin_recalculate_all_levels(request):
```

if I get the `admin_` part correctly.

Reply:
@nbellol / @evildmp also this one please